metrics/histogram.go

```diff
@@ -0,0 +1,112 @@
+package wavefront
```

Comment:
Put this under `histograms/histogram.go` and change package to histogram.

In terms of terminology histogram is different than metrics. They are two different products that Wavefront now offers.